benchexec/tablegenerator/react-table/package.json

```diff
@@ -123,12 +157,14 @@
     "file-loader": "^4.2.0",
     "fs-extra": "^8.1.0",
     "html-webpack-plugin": "^4.0.0-beta.8",
+    "husky": "^4.2.3",
```

Comment:
Isn't `husky` more a development tool that one would install locally on your machine (like an editor) instead of per project?

Reply:
Not really as husky is a tool for git hooks and therefore must/should be present whenever we run git commands. 
The configuration for said hooks are defined in the ´package.json´ and might be husky version specific. 
It is therefore recommended to install husky as a dev dependency to enforce its presence and version.

Reply:
Hm, I am not yet convinced about the value of this. In the current config, it only runs a subset of our checks, so developers still need to remember to run other checks manually. But running more checks automatically would soon start to annoy developers if it takes too much time everytime they want to create a temporary commit etc. Furthermore, the JS project is only a part of the repo, and running JS checks on commits that do not touch the JS code is useless.

So I suggest we drop this and developers who like git hooks can of course just configure them locally on their own without much disadvantage.

Reply:
In the current configuration it would run `lint:fix` and then would add the linted files to the commit stage again. 
This would happen automatically for the extensions specified in the `lint:fix` script (namely css, scss, json, js). 
It would also be possible to configure the hook in way where it would only run on files in the `react-tables` subfolder. 
This was mainly an idea to make sure that the ci doesn't fail because of a missing trailing comma or a missing space after open brackets. 
I do however strongly agree on the fact that the JavaScript part of this project is not the main focus and therefore may have implications on people not working with this (sub)module. 
I will remove the hook, and clean up the whole branch.